<launch>
  <env name="ROSCONSOLE_CONFIG_FILE" value="$(find snap_circuits)/rosconsole.conf"/>

  <!-- xml file containing the cells of the board -->
  <param name="snap_circuits/svg_folder" value="$(find snap_circuits)/lib/svg/" />

  <node name="tester" pkg="snap_circuits" type="tester" respawn="false" output="screen"/>

</launch>
